exonum-light-client/src/test/java/com/exonum/client/ExplorerApiHelperTest.java

```diff
@@ -113,7 +114,115 @@ void parseGetTxResponseCommitted() {
   }
 
   @Test
-  void parseGetTxResponseCommittedWithError() {
+  void parseGetTxResponseCommittedWithPanic() {
+    TransactionMessage expectedMessage = createTransactionMessage();
+    String errorDescription = "Unexpected error";
+    String json = "{\n"
+        + "    'type': 'committed',\n"
+        + "    'content': {\n"
+        + "        'debug': {\n"
+        + "            'amount': 1,\n"
+        + "            'seed': 5019726028924803177\n"
+        + "        },\n"
+        + "        'message': '" + toHex(expectedMessage) + "'\n"
+        + "    },\n"
+        + "    'location': {\n"
+        + "        'block_height': 1,\n"
+        + "        'position_in_block': 0\n"
+        + "    },\n"
+        + "    'location_proof': {\n"
+        + "        'val': 'e8a00b3747d396be45dbea3bc31cdb072'\n"
+        + "    },\n"
+        + "    'status': {\n"
+        + "        'type': 'panic',\n"
+        + "        'description': \"" + errorDescription + "\""
+        + "    }\n"
```

Comment:
This is the only variable thing in this and the following tests. It seems like the main and/or test code have to be refactored to not duplicate the rest needlesly. E.g.,

1. A template JSON for testing various results + a source of (JSONs, expected execution status) + a parameterized test asserting on the expected status (and the constants, if needed).
2. A modification of main code that allows to test the various results independently from the transaction response.